test/testinput/generate_aircraft_bias_csv_test.py

```diff
@@ -0,0 +1,79 @@
+# (C) Copyright 2023 NOAA/NWS/NCEP/EMC
```

Comment:
Instead of this nearly duplicate test py script, what about running the src/ script instead?

Reply:
I think that's the way to go, less code to maintain in the future going forward.

Reply:
thanks for the comments @CoryMartin-NOAA ,,  

@BrettHoover-NOAA will watch for the changes recommended thanks

Reply:
fixed